src/services/reviews.js

```diff
@@ -761,9 +761,22 @@ async function handleComments(comments, resourceId, userId) {
 		// Separate comments into ones that need to be updated and ones that need to be created
 		const commentsToUpdate = []
 		const commentsToCreate = []
+		// parant id = null --> parent
+		// parant id = 123 --> child
 
 		for (let comment of comments) {
+			if (comment?.parent_id) {
+				const isCommentValid = await isParantCommentValid(comment.parent_id, resourceId)
```

Comment:
@adithyadinesh0412 why findone in loop ? we should fetch all the comment at once